.gitignore

```diff
@@ -1 +1,5 @@
 *.pyc
+/demos/.vs
```

Comment:
This is redundant, because you already have `.vs/` listed below.

Reply:
gitignore changes are removed